src/outputs/outputs_main.py

```diff
@@ -37,17 +36,29 @@ def run_output(
     NETWORK_OR_HDFS = config["global"]["network_or_hdfs"]
     output_path = config[f"{NETWORK_OR_HDFS}_paths"]["output_path"]
 
-    # Create combined ownership column using mapper
-    estimated_df = combine_dataframes(estimated_df, ultfoc_mapper)
+    # Prepare the columns needed for outputs:
 
-    # add cora status "form_status" using mapper
-    estimated_df = create_cora_status_col(estimated_df, cora_mapper)
+    # Join foriegn ownership column using ultfoc mapper
+    estimated_df = map.join_fgn_ownership(estimated_df, ultfoc_mapper)
 
-    # Creating blank columns for short form output
-    short_form_df = run_shortform_prep(estimated_df, round_val=4)
+    # Map to the CORA statuses from the statusencoded column
+    estimated_df = map.create_cora_status_col(estimated_df, cora_mapper)
+
+    # Map the sizebands based on frozen employment
+    estimated_df = map.map_sizebands(estimated_df)
+
+    # Map the itl regions using the postcodes
+    estimated_df = map.join_itl_regions(estimated_df, postcode_itl_mapper)
+
+    # Prepare the shortform output dataframe
+    short_form_df = short.run_shortform_prep(estimated_df, round_val=4)
+
+    # Create short form output dataframe with required columns from schema
+    schema_path = "src/outputs/output_schemas/frozen_shortform_schema.toml"
```

Comment:
Should this path be in the config file?

Reply:
Good point. We don't have any of our toml schemas in our developer config yet, so i've added now a new section called schema_paths.